src/views/Suites.vue

```diff
@@ -116,6 +117,12 @@ export default {
   },
   beforeCreate() {
     suiteService.getSuites()
+    this.polling = setInterval(() => {
+      suiteService.getSuites()
+    }, 5000)
```

Comment:
Comment on another time interval that will be replaced by WebSocket.

Reply:
+1